lib/util.js

```diff
@@ -81,4 +80,12 @@ function defineOperation(fn, paramDefs) {
   };
 }
 
-module.exports = { defineOperation, validateParameter };
+function loadBindings() {
+  try {
+    return require('../build/Release/kerberos.node');
+  } catch {
+    return require('../build/Debug/kerberos.node');
+  }
+}
```

Comment:
is the version of the build that's available just determined by how node-gyp is invoked? I'd assume that we build the release version by default, but just want to confirm.